GenderPayGap.WebUI/Controllers/Account/ChangeEmailController.cs

```diff
@@ -72,6 +72,9 @@ public IActionResult ChangeEmailPost(ChangeEmailViewModel viewModel)
             }
 
             SendVerificationEmail(viewModel.NewEmailAddress, user);
+            user.NewEmailAddress = viewModel.NewEmailAddress;
+            user.NewEmailAddressRequestDate = VirtualDateTime.Now;

```

Comment:
Is VirtualDateTime a .net thing or a homemade class?

Reply:
A homemade class. It is used to initialize several other DateTime objects.